### PR TITLE
[DeleteFilesV1] Added new argument that change work behavior with files starting with a dot.

### PR DESCRIPTION
**Pull request by vladislav-ryzhov**
**Task name**: DeleteFilesV1

**Description**: Added new input and new condition for new task behavior.  If new input checked, add "dot: true" option to match options object.

**Documentation changes required:**  Y

**Added unit tests:** Y

**Attached related issue:** #9457

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DeleteFilesV1/task.json [hunk 1]
```diff
@@ -45,6 +52,15 @@
             "defaultValue": false,
             "required": false,
             "helpMarkDown": "Attempt to remove the source folder as well."
+        },
+        {
+            "name": "DeleteFilesWithDot",
```
Comment (egor-bryzgalov) [inline Tasks/DeleteFilesV1/task.json:57]: nit: I'd like to suggest renaming this to `removeDotfiles`

**Review by egor-bryzgalov — APPROVED** [review]
Looks great! Thank you!

**Review by DaniilShmelev — APPROVED** [review]
Take a look at the comments, but other than that LGTM!

File: Tasks/DeleteFilesV1/Tests/L0.ts [hunk 2]
```diff
@@ -89,6 +89,46 @@ describe('DeleteFiles Suite', function () {
         }, tr, done);
     });
 
+    it('Deletes files starting with a dot', (done: Mocha.Done) => {
+
```
Comment (anatolybolshakov) [inline Tasks/DeleteFilesV1/Tests/L0.ts:93]: Nit: redundant space

**Review by anatolybolshakov — APPROVED** [review]

**Pull request merged**